plugins/catalog-backend/src/ingestion/processors/ldap/config.ts

```diff
@@ -109,6 +109,15 @@ export type GroupConfig = {
     // The name of the attribute that shall be used for the value of the
     // spec.type field of the entity. Defaults to "groupType".
     type: string;
+    // The name of the attribute that shall be used for the value of the
+    // spec.profile.displayName field of the entity. Defaults to "cn".
+    displayName: string;
+    // The name of the attribute that shall be used for the value of the
+    // spec.profile.email field of the entity.
+    email: string;
```

Comment:
make nullable or document the default in the comment?

Reply:
Nice that you spotted that! Fixed!